protocol/avdl/keybase1/notify_fs.avdl

```diff
@@ -20,6 +20,9 @@ protocol NotifyFS {
   @lint("ignore")
   void FSSyncStatusResponse(FSSyncStatus status, int requestID) oneway;
 
+  @lint("ignore")
+  void FSOverallSyncStatusChanged(FSOverallSyncStatus status) oneway;
```

Comment:
Is there a chance in the future that you'll need a notification about the local disk usage as well, to display the "out of space" warning?  If so, then I think using a `FolderSyncStatus` here would be better (instead of defining a new type) because it also gives the local disk stats.

Reply:
@songgao

Reply:
switched to `FolderSyncStatus`.

Reply:
Sorry didn't check email/GH last couple days. Sounds good to me!